rdf4j/src/main/scala/org/w3/banana/rdf4j/io/Rdf4jRDFReader.scala

```diff
@@ -0,0 +1,72 @@
+package org.w3.banana.rdf4j.io
+
+import java.io._
+import java.util.LinkedList
+
+import org.eclipse.rdf4j.model._
+import org.eclipse.rdf4j.model.impl.{LinkedHashModel, LiteralImpl, StatementImpl}
+import org.w3.banana._
+import org.w3.banana.io._
+import org.w3.banana.rdf4j.Rdf4j
+
+import scala.util._
+
+trait CollectorFix extends org.eclipse.rdf4j.rio.helpers.StatementCollector {
+
+  def ops: RDFOps[Rdf4j]
+
+  override def handleStatement(st: Statement): Unit = st.getObject match {
+    case o: Literal if o.getDatatype == null && o.getLanguage == null =>
+      super.handleStatement(
+        new StatementImpl(
+          st.getSubject,
+          st.getPredicate,
+          new LiteralImpl(o.getLabel, ops.xsd.string)))
```

Comment:
This class is deprecated. The documentation points to using `ValueFactory#createLiteral(String, IRI)` instead.
So is `StatementImpl`.

Btw. the `CollectorFix` method name indicates to me that it was there to try to fix a bug. It may be that with the new Rdf4J that is no longer needed. If it works it is fine, but it may be worth putting up an issue for that, for a later todo, if you don't have time right now.